docs/overview.md

```diff
@@ -0,0 +1,72 @@
+SecureDrop Overview
+===================
+
+SecureDrop is an open-source whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. It was originally created by the late Aaron Swartz and is currently managed by [Freedom of the Press Foundation][FPF].
+
+Infrastructure
+--------------
+
+![SecureDrop architecture overview diagram](./SecureDrop_diagram.png)
+
+There are four main components of SecureDrop: the servers, the administrators, the sources, and the journalists.
+
+
+### Servers
+
+At SecureDrop's heart is a pair of severs: the *Application (“App”) Server* which runs the core SecureDrop software, and the *Monitor (“Mon”) Server* which keeps track of the *Application Server* and sends out alerts if there's a problem. These two servers run on dedicated hardware connected to a dedicated firewall appliance. They are typically located physically inside the newsroom.
+
+
+### Systems Administrator
+
+The SecureDrop servers are managed by a systems administrator; for larger newooms, there may be a team of systems administrators. The administrator uses a dedicated *Admin Workstation* running [Tails]() and connects to the App and Mon servers over authenticated [Tor Hidden Services]() and manages them using [Ansible]().
```

Comment:
To use the "reference-style"/"implicit link name" links in Markdown, you need to follow the link name with `[]`, not `()`. Using `()` just results in links with an empty `href=""` attribute, so they just point to the page you're already on.